### PR TITLE
Proofreading

### PR DESCRIPTION
**Pull request by phillipdwright**
Just a few suggestions!

### CODE REVIEW

File: unit-2-functions/lesson-1-introduction-to-functions/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ They usually receive data as input and `return` a result as output.
 
 Functions are built like this:
 
-```
+```python
```
Comment (phillipdwright) [inline unit-2-functions/lesson-1-introduction-to-functions/README.md]: I think this code block and the other one in this file in particular would read much better with Python syntax highlighting.  I feel like that will help the reader distinguish more easily between the lines that are comments and the lines that are executing code.

File: unit-2-functions/lesson-1-introduction-to-functions/README.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ Functions are great!
 
 They are like bite-sized programs in your code that can be executed to perform operations for you. You can combine them or reuse them in your code to do some pretty wonderful things!
 
-The usually receive data as input and `return` a result as output.
+They usually receive data as input and `return` a result as output.
```
Comment (phillipdwright) [inline unit-2-functions/lesson-1-introduction-to-functions/README.md]: This addresses issue https://github.com/rmotr-curriculum/free-python-course/issues/1.

File: unit-2-functions/lesson-1-introduction-to-functions/README.md [hunk 3]
```diff
@@ -17,27 +17,27 @@ def my_function(parameter1, parameter2):
...
 
 # Or you can give literal arguments and just send numbers instead of variables
 function_result = my_function(1, "bob")
 ```
 
-You use the functions name, and pass it arguments (function inputs). They are called arguments in a function call and parameters in a function definition. Note the difference.
+You use the function's name, and pass it arguments (function inputs). They are called arguments in a function call and parameters in a function definition. Note the difference.
 
-These arguments are sent to the function defintion and are saved using the parameter names. When we run the first function call below, argument1 is stored in a variable called parameter1 and argument2 is stored in a variable called parameter2 in the function definition when it runs.
+These arguments are sent to the function definition and are saved using the parameter names. When we run the first function call below, argument1 is stored in a variable called parameter1 and argument2 is stored in a variable called parameter2 in the function definition when it runs.
```
Comment (phillipdwright) [inline unit-2-functions/lesson-1-introduction-to-functions/README.md]: This addresses issue https://github.com/rmotr-curriculum/free-python-course/issues/2.

**Comment by santiagobasulto** [comment]
Oh Phil, I didn't see these PRs and now they have conflicts. I'll manually check the diffs and fix it locally. Thanks so much!

**Comment by santiagobasulto** [comment]
Fixed in 0f9e608!

**Pull request closed without merging**